lib/build.gradle

```diff
@@ -42,7 +42,14 @@ dependencies {
     api "com.cloudinary:cloudinary-core:${cloudinaryLibsVersion}"
 
     implementation "com.android.support:support-compat:${androidSupportVersion}"
-    implementation('com.evernote:android-job:1.2.6', {
+    implementation ('android.arch.work:work-runtime:1.0.0', {
```

Comment:
@nitzanj , you decided to include the work manager eventually? 
what's the reason?